src/test/java/seedu/parser/ParserTest.java

```diff
@@ -0,0 +1,180 @@
+
+import org.junit.jupiter.api.Test;
+import seedu.data.Book;
+import seedu.data.SysLibException;
+
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertArrayEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+class ParserTest {
+
+    @Test
+    public void testProcessExitCommand() {
+        Parser parser = new Parser();
+        String validResponse = "exit";
+
+        ByteArrayOutputStream outputStream = new ByteArrayOutputStream();
+        System.setOut(new PrintStream(outputStream));
+        parser.process(validResponse);
+
+        System.setOut(System.out);
+        String output = outputStream.toString();
+
+        String expectedOutput = "Bye, hope to see you again soon!"+ System.lineSeparator()+
+                "____________________________________________________________"+System.lineSeparator();
+        assertEquals(expectedOutput, output);
+    }
+
+    @Test
+    public void testProcessHelpCommand() {
+        Parser parser = new Parser();
+        String validResponse = "help";
+
+        ByteArrayOutputStream outputStream = new ByteArrayOutputStream();
+        System.setOut(new PrintStream(outputStream));
+        parser.process(validResponse);
+
+        System.setOut(System.out);
+        String output = outputStream.toString();
+
+        String expectedOutput = "Commands available:" + System.lineSeparator()+
+                "add: add an item (e.g. add /id ID /t TITLE /a AUTHOR /tag TAG /i ISBN [/g GENRE])" +
+                System.lineSeparator()+
+                "delete: delete an item (e.g. delete /id 123456789)" + System.lineSeparator()+
+                "list: (e.g. list /tag B /g Fiction /a J. K. Rowling /i 9780763630188)" + System.lineSeparator() +
+                "find: (e.g. find /i 9780763630188)" + System.lineSeparator() +
+                "exit: (e.g. exit)" + System.lineSeparator() +
+                "For more information, please refer to our user guide at:" +
+                "https://ay2324s1-cs2113t-w11-1.github.io/tp/UserGuide.html" + System.lineSeparator() +
+                "____________________________________________________________" + System.lineSeparator();
+
+        assertEquals(expectedOutput, output);
+    }
+
+    @Test
+    public void testProcessUnknownCommand() {
+        Parser parser = new Parser();
+        String validResponse = "bye";
+
+        ByteArrayOutputStream outputStream = new ByteArrayOutputStream();
+        System.setOut(new PrintStream(outputStream));
+
+        parser.process(validResponse);
+
+        System.setOut(System.out);
+        String output = outputStream.toString();
+
+        String expectedOutput = "no commands found. Enter \"help\" for a list of commands." + System.lineSeparator() +
+                "____________________________________________________________" + System.lineSeparator();
```

Comment:
Your output has no line divider " "____________________________________________________________"  but your expecting a line divider.
![image](https://github.com/AY2324S1-CS2113T-W11-1/tp/assets/84222810/4855b7c8-0bcf-4e0b-800f-75d0f1e389cd)

Reply:
I have already made that change...
<img width="1430" alt="Screenshot 2023-10-16 at 4 16 54 PM" src="https://github.com/AY2324S1-CS2113T-W11-1/tp/assets/142559789/47a45b54-07df-4616-868a-2850c0e435d7">